keyboards/aidansmithdotdev/fine40/fine40.h

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
This file should be removed, as matrix data is already provided for layouts in `info.json`

Reply:
Removed